charts/px-backup/templates/_helpers.tpl

```diff
@@ -0,0 +1,56 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "pxcentral-onprem.name" -}}
```

Comment:
There are still a lot of references to `pxcentral-onprem`, need to replace that with px-backup

Reply:
replaced with px-backup